tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -399,7 +450,47 @@ void test_various_containers() {
     test_one_container<list<int>>(); // bidi, not vectorizable
 }
 
+#if _HAS_CXX20
+constexpr bool test_constexpr() {
+    int data[] = {2, 1, 3, 3, 3, 3, 4, 6, 5};
+
+    assert(count(begin(data), end(data), 3) == 4);

```

Comment:
In test code, I recommend following the principle "unique numbers for unique purposes". This makes it a lot easier to understand what various values mean. Here, because the test data uses small integers, there's visual ambiguity between element values like `3`, counts of elements like `4`, and distances like `2`.

If the test data were simply multiples of 10, then `30` would obviously be an element, making the counts like `4` and distances like `2` visually distinctive.